content/blog/2024/06/kubewarden-1-14-release.md

```diff
@@ -0,0 +1,112 @@
+migration of VAP policies to Kubewarden, enabling users to leverage the
+benefits of running these policies within the Kubewarden framework. For a
+detailed guide on how to use this new feature, check out our
+[documentation](https://kubewarden.io/docs).
+
+## Policy Fixes
+
+In this release, we have addressed two issues in our existing policies:
+
+1. **[Environment Variable
+   Policy](https://github.com/kubewarden/environment-variable-policy/issues/13)**:
+   Previously, variables with values were not blocked if only the name was
+   specified in the settings. We have updated the validation code to ensure
+   that the policy will validate the names even when the user does not define
+   the envvar value in the rule.
+   
+2. **[Container Resources
+   Policy](https://github.com/kubewarden/container-resources-policy/issues/33)**:
+   Users faced issues when trying to create policies with only CPU settings
+   empty memory values. The policy settings were sent with empty values,
+   causing validation errors. We have fixed this by allowing a single resource
+   setting to be empty, resolving the issue without breaking policy usage.
+
+## Best Practices: Domain-Qualified Finalizers
+
+The Kubewarden controller now follows best practices by using domain-qualified
+finalizers. Users must be aware that if they downgrade a Kubewarden
+installation, the old controller will not be able to identify the new
+finalizer. Leaving the cluster in a state where the resource could not be
+removed requiring manual intervention. 
+
+NOTE: Downgrade procedures are not supported by Kubewarden project. See the
+[documentation](https://docs.kubewarden.io/reference/upgrade-path#downgrades)
+to learn more about that
+
+## KCD Italy: CEL for Cluster Security
+
+One of the Kubewarden maintainers,  [Flavio
+Castelli](https://github.com/flavio/),  delivered an insightful talk at [KCD
+Italy](https://community.cncf.io/events/details/cncf-kcd-italy-presents-kcd-italy-2024/)
+on "How to leverage and extend CEL for your cluster security." He explored the
+capabilities and trade-offs of the CEL-based admission controller introduced in
+Kubernetes 1.28 and discussed how to use CEL with other dynamic admission
+controllers. This talk highlighted the practical applications and advantages of
+using CEL for cluster security.
+
+## Known Bugs and Future Plans
+
+During the release process, we identified a bug related to the policy server
+certificate rotation. While the Root CA is set to expire in 10 years, each
+policy-server cert secret has a one-year expiry. But the controller is
+currently unable to renew them automatically. In this release, we will ensure
+that policy-server secrets are created with a 10-year expiry. And in future
+releases we plan to properly implement an automated renewal process. For now,
+users can manually delete the expired cert secret (`policy-server-default`) and
+trigger the controller reconciliation by adjusting the replicas.
```

Comment:
```suggestion
trigger the controller reconciliation by adding/removing/updating a policy or by adjusting the number of replicas of a `PolicyServer`.
```

We are going to work on a better solution to this problem in an upcoming release of Kubewarden.